### ISSUE TITLE
Should we consider XDM as dead ?

### ISSUE THREAD

**Issue by realgooseman** [subhra74/xdm#1218]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

After nearly a year without updates and no signs of activity from the developer, should we consider XDM as dead ?

For users close to @subhra74, did something happen to him in his personal life or did he just lose interest ?

In both cases, it's too bad because it was the only FOSS download manager that could compete with IDM and FDM. There's Motrix too which looks like it isn't actively maintained anymore as well.

**Comment by Ktosnapewno999** [comment]
i don't no did you seen but, he updated  Indonesian.txt 10 months ago 
<img width="1617" alt="Screenshot 2024-04-10 at 10 20 05" src="https://github.com/subhra74/xdm/assets/159190935/ff97effc-4893-4973-a9ec-a8857050e218">

**Comment by realgooseman** [comment]
Yes I've seen it, that's why I said "After **nearly a year** without updates and no signs of activity from the developer".
Very unfortunate that he didn't leave any announcements or put the project in "maintenance mode".

**Comment by mahdihasan2006** [comment]
Indeed he has been inactive in GitHub all together. Also he he has put a 😷 emoji beside his GitHub profile picture. I think something has happen to him.

**Comment by realgooseman** [comment]
He created a repo a few days ago, good news. Let's hope for some clarification on XDM's future soon.

**Comment by rollingmoai** [comment]
- Duplicate of #638?
